internal/rset/tx.go

```diff
@@ -0,0 +1,66 @@
+package rset
+
+import (
+	"database/sql"
+	"github.com/nalgeon/redka/internal/core"
+	"github.com/nalgeon/redka/internal/sqlx"
+	"time"
+)
+
+var sqlAdd = []string{`
```

Comment:
Please use consts (see `rzset` for reference).

Reply:
OK. I saw a slice of queries in another command. I referred it from there, but `sqlAdd1` and `sqlAdd2` make sense.